jobs-data-modelling_en.md

```diff
@@ -9,13 +9,11 @@ As Research Associate you will work in collaboration with project partners at th
 Your responsibilities will include:
 
 * Authoring a doctoral dissertation in informatics (computer science)
-* Research on the representation of linked information assets using semantic technologies
-* Versioning of heterogeneous linked information assets (e.g., in notations, annotations, recordings, and their relationships)
-* Establishing solutions to the modelling of versioned user-generated data (annotations, audio recordings) and their relationships to other project data (notations, transcriptions)
-* Research on the extension of the MEI standard toward modelling music notations in tablature-based formats
+* Research on the representation of linked information assets using semantic technologies with a special focus on integrating user generated data (annotations, audio-recordings) into the project context
+* Research on a sustainable versioning scheme for heterogeneous linked information assets coming from the project (e.g., in notations, annotations, audio-recordings, and their relationships)
+* Assistance during the extension of the MEI standard toward enabling modelling music notations in tablature-based formats with MEI
```

Comment:
"Collaborative research into the extension of the MEI standard toward modelling music notations in tablature-based formats" ?